### PR TITLE
Add Updater auto

### PR DESCRIPTION
**Pull request by unanoc**
Implemented Update auto logic for BEP2.

### CODE REVIEW

File: configs/example.config.yaml [hunk 1]
```diff
@@ -2,7 +2,9 @@ app:
   log_level: info
 
 client_urls:
-  binancedex: "https://dex-atlantic.binance.org"
+  binance:
+    explorer: https://explorer.binance.org
```
Comment (vikmeup) [inline configs/example.config.yaml:6]: is there a way to ditch `explorer` endpoint and use just `dex?

Comment (unanoc) [reply configs/example.config.yaml:6]: i will figure out

Comment (unanoc) [reply configs/example.config.yaml:6]: there's no way

File: pkg/files.go [hunk 2]
```diff
@@ -23,3 +30,73 @@ func IsFileAllowedInPR(path string) bool {
...
+
+	err = ioutil.WriteFile(path, file, 0644)
+	if err != nil {
+		return fmt.Errorf("failed to write json to file: %v", err)
+	}
+
+	return nil
+}
+
+func ReadJSONFile(path string, payload interface{}) error {
```
Comment (tw-covain) [inline pkg/files.go]:
```suggestion
func ReadJSONFile(path string, result interface{}) error {
```

**Review by tw-covain — APPROVED** [review]

File: src/core/updaters.go [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+)
+
+const (
+	assetsPage       = 1
+	assetsRows       = 1000
+	marketPairsLimit = 1000
+	tokensListLimit  = 10000
+
+	twLogoURL       = "https://trustwallet.com/assets/images/favicon.png"
+	assetsURI       = "https://assets.trustwalletapp.com"
```
Comment (vikmeup) [inline src/core/updaters.go]: add this to config

File: src/core/updaters.go [hunk 4]
```diff
@@ -0,0 +1,259 @@
...
+	"github.com/trustwallet/go-primitives/types"
+)
+
+const (
+	assetsPage       = 1
+	assetsRows       = 1000
+	marketPairsLimit = 1000
+	tokensListLimit  = 10000
+
+	twLogoURL       = "https://trustwallet.com/assets/images/favicon.png"
```
Comment (vikmeup) [inline src/core/updaters.go:28]: why do we need `twLogoURL` ?

File: src/processor/service.go [hunk 5]
```diff
@@ -68,6 +66,21 @@ func (s *Service) RunSanityCheck(paths []string) error {
...
+	for _, updater := range updaters {
+		log.WithField("Name", updater.Name).Debug("Running updater")
+
+		err := updater.Run()
+		if err != nil {
+			log.WithError(err).Error()
+		}
+	}
+
+	return nil // ???
```
Comment (vikmeup) [inline src/processor/service.go]: Is this a todo?

**Review by vikmeup — APPROVED** [review]

File: src/core/updaters.go (hunk 4 above)
Comment (unanoc) [reply src/core/updaters.go:28]: i assume it's used somewhere in the app? Is tokenlist.json used by the app? if it's just some info - I can remove it

**Pull request merged**